resources/ext/ezbake.conf

```diff
@@ -2,8 +2,8 @@ ezbake: {
   pe: {}
   foss: {
     redhat: { dependencies: ["puppet >= 3.7.3", "puppet < 5.0.0"],
```

Comment:
Also, redhat doesn't support dependency disjunction, but debian does. So there is no way to say we depend on puppet or puppet-agent on redhat.